foglamp/FogLAMP
# FOGL-1487 - Create a addScheduledTask REST entry point that is similar to addService

Steps to test:
1. make clean, make, scripts/foglamp reset
2. Start server.
3. Check scripts/foglamp status + syslog that NO 'HTTP North' process is running.
4. In another folder, download, extract and run
[ldeploy-http-north-plugin.sh.zip](https://github.com/foglamp/FogLAMP/files/2191759/ldeploy-http-north-plugin.sh.zip). Before executing this script, please ensure that foglamp-north-http/cmds-http-north.sql contains only below commands:
```
-- SQLite only
-- Plugin loader configuration
INSERT INTO destinations ( id, description)
    VALUES ( 2, 'HTTP North' );

INSERT INTO streams ( id, destination_id, description, last_object )
    VALUES ( 3, 2, 'HTTP North', 0 );

-- Statistics
INSERT INTO statistics (key, description, value, previous_value)
    VALUES('SENT_3', 'Readings data sent via HTTP North', 0, 0);

```
5. Check scripts/foglamp status + syslog that now 'HTTP North' process is running.
6. We may also check the actual sending process by ingesting data via, say fogbench, and see if the sending process via HTTP North is actually working. 

Test result:
`=================================================== 1158 passed, 31 skipped in 84.67 seconds ====================================================
`
